### DISCUSSION TITLE
I can't press the checkbox list.

### DISCUSSION THREAD

**Discussion by yamanoku** [community/community#8655] [Other Feature Feedback, Questions, & Ideas]
Unable to click on the Star list when it is displayed on the top page of the repository.
[![](https://i.gyazo.com/57f667b7844de96c0051779d856c11a2.gif)](https://gyazo.com/57f667b7844de96c0051779d856c11a2)

**Comment by brianlovin** [comment]
Thanks for the heads up, I've filed an issue for our team to investigate what could be going wrong here.

**Comment by rmathew1992** [comment]
Hello! I've started to investigate what might be going on here, and was hoping for some additional info. Could you let me know:

1. What browser you're using?
2. And what device this is on?

Thanks so much for your help!

**Comment by yamanoku** [comment]
@rmathew1992 This is the environment I saw myself in:

1. Brave（Version 1.32.115 Chromium: 96.0.4664.93 (Official Build) (x86_64)）
2. MacBook Pro (13-inch, 2020, Two Thunderbolt 3 ports) macOS Big Sur v11.6
